spec/schema.rb

```diff
@@ -271,6 +280,11 @@ class AllowedPlace < ActiveRecord::Base
   multi_tenant :account
 end
 
+class CompositeKeyModel < ActiveRecord::Base
+  multi_tenant :account
+  self.primary_key = %i[account_id entity_id version]
```

Comment:
Based on `primary_key: %i[entity_id version]`, here also should be:
```suggestion
  self.primary_key = %i[entity_id version]
```